core/templates/dev/head/domain/question/pretest-question-backend-api.service.spec.ts

```diff
@@ -109,13 +155,13 @@ describe('Pretest question backend API service', function() {
 
       $httpBackend.expect(
         'GET', '/pretest_handler/expId?story_id=storyId&cursor=').respond(
-        sampleDataResults);
+        responseDictionaries);
       PretestQuestionBackendApiService.fetchPretestQuestions(
         'expId', 'storyId').then(successHandler, failHandler);
       $httpBackend.flush();
 
       expect(successHandler).toHaveBeenCalledWith(
-        sampleDataResults.pretest_question_dicts);
+        sampleDataResultsObjects.pretest_question_objects);
```

Comment:
Since you are only using `pretest_question_objects` in `sampleDataResultsObjects` and using it directly, I think there is no need to have a  `sampleDataResultsObjects` var at all? 
Also, rename pretest_question_objects to camel case (pretestQuestionObjects), since this doesn't represent the return list anyway.

Reply:
I agree. I have pushed these changes.